examples/next/docs/vue3/demo/src/renderers/progressBar.ts

```diff
@@ -0,0 +1,30 @@
+import { baseRenderer } from 'handsontable/renderers';
+import { getRangeValue } from '../utils';
+import { MESSAGE } from '../constants';
+
+const minAllowedValue = 0;
+const maxAllowedValue = 100;
+
+export const progressBarRenderer: typeof baseRenderer = (
+  instance,
+  td,
+  row,
+  column,
+  prop,
+  value,
+  cellProperties
+) => {
+  if (!value) {
+    value = 0;
+  }
+
+  const isValid = /^\d+$/.test(value.toString());
```

Comment:
Instead of creating a second-ish validator, you can use the native Handsontable validator.

To do that, you'd need to set the type of the column (in this case `type="numeric"`) and implement logic similar to the one I suggested in your PR updating the React demo:
https://github.com/handsontable/handsontable/pull/9935#discussion_r1108212050